### ISSUE TITLE
externalize startup script

### ISSUE THREAD

**Issue by ssm2017** [videodromm/netdromm#13]
on my opinion, the startup script should be located on another repo because it is not concerning the code itself

**Comment by brucelane** [comment]
right! and this repo should contain the docker composer files to use the several containers for the 3 repos: netdromm, videodromm-web, videodromm-browser
